kratos/includes/gid_gauss_point_container.h

```diff
@@ -92,15 +92,15 @@ class GidGaussPointsContainer
     }
 
 
-//            virtual void PrintResults( Variable<array_1d<double,3> > rVariable, ModelPart& rModelPart,
-//                                        double SolutionTag, unsigned int ValueIndex )
+//            virtual void PrintResults( Variable<array_1d<double,3> > rVariable, const ModelPart& rModelPart,
+//                                        const double SolutionTag, const unsigned int ValueIndex )
 
     virtual void PrintFlagsResults(
         GiD_FILE ResultFile,
-        Kratos::Flags rFlag,
-        std::string rFlagName,
-        ModelPart& rModelPart,
-        double SolutionTag
+        const Kratos::Flags& rFlag,
```

Comment:
Be careful, you are changing the signature of a `virtual` function
If someone overrides this it will fail